src/main/java/org/broadinstitute/hellbender/engine/filters/WellformedReadFilter.java

```diff
@@ -7,7 +7,7 @@
 
 /**
  * Tests whether a read is &quot;well-formed&quot; -- that is, is free of major internal inconsistencies and issues that could lead
- * to errors downstream. If a read passes this filter, the rest of the hellbender engine should be able to process it without
+ * to errors downstream. If a read passes this filter, the rest of the engine should be able to process it without
```

Comment:
Yes that would be very valuable, thank you. I believe we had done that for MalformedReadFilter in GATK3